pop-api/src/v0/mod.rs

```diff
@@ -16,3 +22,22 @@ impl From<StatusCode> for Error {
 		value.0.into()
 	}
 }
+
+/// Helper method to build `ChainExtensionMethod``
+pub fn build_extension_method_v0(
```

Comment:
As above, can remove pub and add arguments doc comments. Same with the other two functions below.